apps/server/src/modules/federal-state/domainobject/county.do.ts

```diff
@@ -0,0 +1,25 @@
+export interface CountyProps {
+	name: string;
+	countyId: number;
+	antaresKey: string;
+}
+
+export class CountyDO {
+	protected props: CountyProps;
```

Comment:
why protected? should not be private? do we have other class extending this one?